cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/AddBrokersRunnable.java

```diff
@@ -127,6 +127,7 @@ protected OptimizerResult workWithClusterModel() throws KafkaCruiseControlExcept
                                            _executionProgressCheckIntervalMs,
                                            _replicaMovementStrategy,
                                            _replicationThrottle,
+                                           null,
```

Comment:
Why is the `logDirThrottle` argument set to null here? Wouldn't we want new brokers to have the configured logDirThrottle set?